src/search/CMakeLists.txt

```diff
@@ -1,104 +1,984 @@
-cmake_minimum_required(VERSION 2.8.3)
-# For Windows we require CMake 3.12, but this is currently not
-# available for Ubuntu 18.04.
-
-if(NOT FAST_DOWNWARD_MAIN_CMAKELISTS_READ)
-    message(
-        FATAL_ERROR
-        "Run cmake on the CMakeLists.txt in the 'src' directory, "
-        "not the one in 'src/search'. Please delete CMakeCache.txt "
-        "from the current directory and restart cmake.")
-endif()
+# See https://www.fast-downward.org/ForDevelopers/AddingSourceFiles
+# for general information on adding source files and CMake libraries.
+#
+# All libraries are enabled by default and users can disable them by specifying
+#    -DLIBRARY_FOO_ENABLED=FALSE
+# The default behavior can be changed so all non-essential libraries are
+# disabled by default by specifying
+#    -DDISABLE_LIBRARIES_BY_DEFAULT=TRUE
+# In that case, individual libraries can be enabled with
+#    -DLIBRARY_FOO_ENABLED=TRUE
+#
+# Defining a new library:
+#    create_fast_downward_library(
+#        NAME <NAME>
+#        [ DISPLAY_NAME <DISPLAY_NAME> ]
+#        [ HELP <HELP> ]
+#        SOURCES
+#            <FILE_1> [ <FILE_2> ... ]
+#        [ DEPENDS <LIBRARY_NAME_1> [ <LIBRARY_NAME_2> ... ] ]
+#        [ DEPENDENCY_ONLY ]
+#        [ CORE_LIBRARY ]
+#    )
+#
+# <DISPLAY_NAME> defaults to lower case <NAME> and is used to group files
+#   in IDEs and for messages.
+# <HELP> defaults to <DISPLAY_NAME> and is used to describe the cmake option, for example in ccmake.
```

Comment:
Replace line 24 to 26 with 
`# <HELP> is used to describe the cmake option, for example in ccmake.`